app/views/users/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class= "container-fluid">
+
+  <div class= "row">
+    <div class= "col-md-4 col-sm-12 user-info text-center">
+      <%= gravatar_for @current_user %>
+      <p> <%= @current_user.name %> <%= @current_user.lastname %> </p>
+      <p><%= @current_user.email %> </p>
+    </div>
+
+    <div class="col-md-8 post-info text-center">
+    
+    <%= render 'shared/newpost' %>
+    <% if @current_user.posts.any? %>
```

Comment:
- [ ] There shoud be absolutely NO logic in views.
Rails has built-in mechanism to deal with view-related logic which is View Helpers (app/helpers/*_helper.rb). So the common convention is to move any logic in the views to the helpers. Please take a look at [this](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) for more information.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png)